docs/adapters/prisma.md

```diff
@@ -49,6 +49,7 @@ You need to use at least Prisma 2.26.0. Create a schema file in `prisma/schema.p
 datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
+  shadowDatabaseUrl = env("SHADOW_DATABASE_URL") // Support added in Prisma v2.17.0. Allows Prisma to migrate when using a cloud provider that doesn't support the creation of new databases, like Heroku.
```

Comment:
I think this hint is very useful here but I'd suggest to make it more clear that this is only needed in cases where a cloud provider like Heroku is being used and otherwise it's optional. Also added a link to our docs to make for people to learn more about the concept of the shadow DB 👍 

```suggestion
  shadowDatabaseUrl = env("SHADOW_DATABASE_URL") // Only needed when using a cloud provider that doesn't support the creation of new databases, like Heroku. Learn more: https://pris.ly/migrate-shadow
```